examples/sticky.ads.amp.html

```diff
@@ -160,7 +159,7 @@
     </div>
   </header>
   <main role="main">
-  <amp-sticky-ad layout="nodisplay">
+  <amp-sticky-ad layout="container">
```

Comment:
Can I see the updated code? The goal is to remove `display:none` and set `visibility:hidden` as soon as possible, e.g. in the `buildCallback`. I can't imagine the blink will still be there in this case.